### ISSUE TITLE
Go mod issues with v4

### ISSUE THREAD

**Issue by Nikola-Milovic** [bold-commerce/go-shopify#278]
```console
> go get github.com/bold-commerce/go-shopify/v4
go: module github.com/bold-commerce/go-shopify@upgrade found (v2.3.0+incompatible), but does not contain package github.com/bold-commerce/go-shopify/v4
```

Same result in both my project and a clean project

**Comment by oliver006** [comment]
Ahhhhh, thanks for letting me know, I'll get that fixed later today.

**Comment by oliver006** [comment]
[Re-released v4.0.0](https://github.com/bold-commerce/go-shopify/releases/tag/v4.0.0)

Thanks again for the heads-up, I forgot to replace that one `v3`

**Comment by rubenv** [comment]
@oliver006 I'm still getting this, as it might be cached somewhere. Would you mind bumping things so there's a truly new version number available?

**Comment by oliver006** [comment]
@rubenv  - A little while back we released [v4.1.0](https://github.com/bold-commerce/go-shopify/releases/tag/v4.1.0)

Is it working now?

**Comment by rubenv** [comment]
Yes, many thanks!
